src/harness/harnessGlobals.ts

```diff
@@ -27,22 +27,4 @@ var assert: typeof _chai.assert = _chai.assert;
         }
     };
 }
-
-var global: NodeJS.Global = Function("return this").call(undefined); // eslint-disable-line no-new-func
-
-declare var window: {};
-declare var XMLHttpRequest: new() => XMLHttpRequest;

```

Comment:
where did XHR go?

Reply:
It was 100% unused. So it's _gone_.

Reply:
(Historical: It was previously used in the browser specific parts of the harness that got gutted when we removed the browser test build (since it broke so often with other infra changes, as it wasn't regularly used))